CONTRIBUTING.md

```diff
@@ -33,5 +58,15 @@
 - ➖ `:heavy_minus_sign:` : Deletion of some code
 - ♻️ `:recycle:` : Refactor code
 
-# Extra
-- 🚀 `:rocket:` : Final commit
\ No newline at end of file
+### Extra
+- 🚀 `:rocket:` : Final commit
+
+## Releases
+
+Here we will explain how the files located in the [release page](https://github.com/Tugduoff/RType/releases) of Github are  produced
```

Comment:
Double space between `are` and `producted`

Reply:
Fixed